ush/parsing_namelists_WW3.sh

```diff
@@ -77,25 +77,15 @@ WW3_namelists(){
 # --------------------------------------------------------------------------- #
 # Create ww3_multi/shel.inp
 
-  if [ $waveMULTIGRID = ".true." ]; then
-    # ww3_multi template
-    if [ -f ${PARMgfs}/wave/ww3_multi.inp.tmpl ]; then
-      cp ${PARMgfs}/wave/ww3_multi.inp.tmpl ww3_multi.inp.tmpl
-    fi
-    if [ ! -f ww3_multi.inp.tmpl ]; then
-      echo "ABNORMAL EXIT: NO TEMPLATE FOR WW3 MULTI INPUT FILE" 
-      exit 11 
-    fi
-  else 
-    # ww3_multi template
-    if [ -f ${PARMgfs}/wave/ww3_shel.inp.tmpl ]; then
-      cp ${PARMgfs}/wave/ww3_shel.inp.tmpl ww3_shel.inp.tmpl
-    fi
-    if [ ! -f ww3_shel.inp.tmpl ]; then
-      echo "ABNORMAL EXIT: NO TEMPLATE FOR WW3 SHEL INPUT FILE" 
-      exit 12
-    fi
+  # ww3_multi template
```

Comment:
Remove this comments that point to ww3_multi, as you are removing this as an option.  Above on line 78, ww3_mutli should be removed and we should simply have the comment "create ww3_shel.inp"